examples/aws-rds-example/Pulumi.yaml

```diff
@@ -0,0 +1,5 @@
+name: rds-example
```

Comment:
Consider adding a quick README and a script how to initialize this (pulumi package add). Since parameterized providers didn't make it into Pulumi.yaml yet this won't work OOTB.